### PR TITLE
Added onAvatarPress

### PR DESCRIPTION
**Pull request by SunnyGurnani**
GiftedAvatar already had onPress but was not being utilized. Added onAvatarPress () function.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ See [example/App.js](example/App.js)
 - **`renderSend`** _(Function)_ - render the send button
 - **`renderAccessory`** _(Function)_ - renders a second line of actions below the message composer
 - **`bottomOffset`** _(Integer)_ - distance of the chat from the bottom of the screen, useful if you display a tab bar
-
+- **`onAvatarPress`** _(Function)_ - Called when Avatar is presses
```
Comment (dgdavid) [inline README.md]: It seems that you have a typo on «presses». It should be «pressed»

**Comment by benjaminb10** [comment]
+1

File: src/Avatar.js [hunk 2]
```diff
@@ -27,6 +27,7 @@ export default class Avatar extends React.Component {
         <View style={[styles[this.props.position].container, this.props.containerStyle[this.props.position]]}>
           <GiftedAvatar
             avatarStyle={StyleSheet.flatten([styles[this.props.position].image, this.props.imageStyle[this.props.position]])}
+            onPress={this.props.onAvatarPress}
```
Comment (cooperka) [inline src/Avatar.js]: We probably don't want `onPress` here because this is actually a blank avatar that isn't visible.

**Comment by cooperka** [comment]
This PR can be closed in favor of https://github.com/FaridSafi/react-native-gifted-chat/pull/427 (rebased).

**Pull request closed without merging**